### DISCUSSION TITLE
Is it possible to make this menu also non-immersive?

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#676] [Ideas]
![Clipboard_20220116](https://user-images.githubusercontent.com/1271736/149641310-cc483b00-a6d8-43f3-84ac-77d8c62a9f79.png)

**Comment by valinet** [comment]
StartAllBack reimplements legacy looking jump lists as far as I can tell. The old ones do not seem to be located in `explorer.exe` anymore, or at least SAB just mostly reimplements it. Maybe @Simplestas can tell you more about it. I don't know more.
